client/components/video-element.js

```diff
@@ -218,15 +216,15 @@ class VideoElement extends HTMLElement {
   scrollIntoViewWithOffset (offset, behavior = 'smooth') {
     const top = this.getBoundingClientRect().top - offset - document.body.getBoundingClientRect().top
     scrollTo({ top, behavior })
-  }  
+  }
 }
 
 customElements.define('video-element', VideoElement)
 
 function tryFormatDate(date) {
   try {
-    return new Date(date).toISOString().substring(0, 10)
+    return new Date(date).toLocaleDateString(navigator.language || 'en-US').substring(0, 10)
```

Comment:
tried to display date more user friendly. but I have only en-us locale set in browser. If any bugs here, feel free to revert